MovieAppPetProject/MovieAppPetProject/GetMovieDetailsByID/MovieDetailsViewController.swift

```diff
@@ -52,10 +51,17 @@ extension MovieDetailsViewController: MovieDetailsViewModelType {
         self.present(alert, animated: true, completion: nil)
     }
 
+    private func updateWatchlistButtonTitle() {
+        if movieDetailsViewModel.isMovieSaved {
+            addToWatchlistButton.setTitle("✓ Saved", for: .normal)
+        }
+    }
+
     private func updateUI() {
         titleLabel.text = movieDetailsViewModel.originalTitle
         overviewLabel?.text = movieDetailsViewModel.overview
-        releaseDateLabel.text = movieDetailsViewModel.releaseDate
+        releaseDateLabel.text = String(movieDetailsViewModel.releaseDate?.prefix(4) ?? "")
```

Comment:
rather don't cast this here, let the function return a string

Reply:
understood. change made